alerts/cluster-network-operator/SouthboundStaleAlert

```diff
@@ -0,0 +1,23 @@
+# Southbound Stale Alert 
+
+## Meaning 
+
+The alert `SouthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
```

Comment:
Same here, if the alert itself is defined based on the timestamp, I think we can remove the `for` in the prometheusrule.